objects/obj_star/Alarm_4.gml

```diff
@@ -1,66 +1,70 @@
 
-var rando,i,contin;
-rando=0;i=0;contin=0;
+// Checks if the planet has industrial facilities to create an ork fleet
+var rando=0,contin=0;
 
-
-repeat(4){
-    i+=1;
+for(var i=1; i<=4; i++){
     // Check for industrial facilities
     if (p_type[i]!="Dead") and (p_type[i]!="Lava"){// Used to not have Ice either
```

Comment:
personally i've found if(!array_contains["dead", "lava", "ice"], p_type[i]){
}
to be a more extendable syntax
but it's all the same really